packages/serverless-contracts/src/features/fullSchema.ts

```diff
@@ -0,0 +1,19 @@
+import { JSONSchema } from 'json-schema-to-ts';
+
+import { getFullContractSchema as getApiGatewayFullContractSchema } from 'contracts/apiGateway/features';
+import { getFullContractSchema as getCloudFormationFullContractSchema } from 'contracts/cloudFormation/features';
+import { ServerlessContract } from 'index';
+
+export const getContractFullSchema = (
+  contract: ServerlessContract,
+): JSONSchema => {
+  switch (contract.contractType) {
+    case 'apiGateway': {
+      // @ts-expect-error FIXME: problem with types here
+      return getApiGatewayFullContractSchema(contract);
+    }
+    case 'cloudFormation': {
+      return getCloudFormationFullContractSchema(contract);
+    }
```

Comment:
Can you add an exhaustiveness check here? https://kimsereylam.com/typescript/2022/02/11/exhaustiveness-checking-in-typescript.html

Reply:
Typescript already throw an error if i remove one of the case
If i had an exhaustive check, the typing of the function will be modified because the default case will not be able to return a fullContractSchema

Reply:
Great then!